decidim-elections/app/models/decidim/elections/election.rb

```diff
@@ -134,6 +134,16 @@ def blocked?
       def allow_resource_permissions?
         true
       end
+
+      # Public: determines if votes can be casted for this election
+      def can_cast_vote?
```

Comment:
When I first saw this I thought that this should be in the permissions system. 

Then I realised that it's not about the user, but about the election. So I thought that it would be better to rename the methods to make them clearer.

But then I realised that we could use existing methods for these cases as we only need to check for `ongoing?` instead of `can_cast_vote?` (we can only see published elections) and for `started?` instead of `can_verify_vote?`. Maybe we can create alias like `voting_period?` or `verifying_period?` if we want to be more explicit.

What do you think about this?

Reply:
Yeah, sounds fair!
My intention was to move the existing logic away from the view and I really just moved it as it was to the model, with bad naming of course 😂 
Yup, I'll go for your suggestion of using `ongoing?` and `started?`